scripts/apps/authoring/authoring/directives/ItemAssociationDirective.ts

```diff
@@ -76,12 +76,14 @@ export function ItemAssociationDirective(renditions) {
 
                 // update item associations on drop
                 elem.on('drop dragdrop', (event) => {
+                    const type = getSuperdeskType(event);
+
                     // drop event should only fire if dragover event is prevented
                     // however, `ng-file-upload` library calls preventDefault on this event
                     // which it shouldn't do, since the element is not controlled by the library.
                     // Because of this, drop is triggered when it shouldn't have been on firefox, but not on chrome.
                     // The same media types check is added from `dragover` method to work around the situation.
-                    if (MEDIA_TYPES.includes(getSuperdeskType(event))) {
+                    if (MEDIA_TYPES.includes(type) || type === 'Files') {
```

Comment:
what about adding `'Files'` to `MEDIA_TYPES`?

Reply:
yep, that makes sense.